### DISCUSSION TITLE
直播间究竟是哪个简化选项导致无法看到超管发的红色警告呢

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4684] [综合讨论 / General]
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/e92ec609-bd10-42e1-9d19-ee24d0c66b92)
![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/1dca942e-fd8e-4051-8d3c-c92bb425be18)
一直看不到超管发的红色SC，但是由于这种机会很稀有，不能逐一排查

**Comment by the1812** [comment]
如果说是 SC 的话, 应该算醒目留言?
不过既然你这么问了那可能这个并不算 SC

**Comment by u2shana** [comment]
> 如果说是 SC 的话, 应该算醒目留言? 不过既然你这么问了那可能这个并不算 SC
俗称红色SC而已，实际是在屏幕中央的红色警告框，像这样的

![image](https://github.com/the1812/Bilibili-Evolved/assets/14146754/d4d5a316-fdf7-4ecf-a065-dfca559a599a)
